capstone/src/main/webapp/youtubeScript.js

```diff
@@ -45,15 +66,16 @@ async function callYoutube() {
       inputCommentsToPerspective([]);
       return;
     }
+    document.getElementById('search-type').innerHTML ="Username Search";
```

Comment:
nit for lines 46, 60, and 69: Let's add a space after the `=`

Reply:
addressed in "spacing and address Daniel's nits"